de.tudarmstadt.ukp.wikipedia.api/src/test/resources/log4j.properties

```diff
@@ -1,6 +1,6 @@
 # Copyright 2016
 # Ubiquitous Knowledge Processing (UKP) Lab
-# Technische Universit�t Darmstadt
+# Technische Universit�t Darmstadt
```

Comment:
Java properties files must be ISO 8859. Possibly the encoding in the repository was UTF-8 and the IDE tried to "fix" that.